source/chapters/effects.rst

```diff
@@ -161,3 +161,24 @@ audience, but this requires a few more steps:
   #. Turn the mix knob right so the effects are heard in the master output
   #. Disable the headphone button for the effect unit so you do not forget to
      turn it off later
+
+Controller Effects Mapping
+==========================
+This section describes how to use the standard Mixxx mapping for effects sections on controllers with the typical layout of 4 knobs (or 3 knobs + 1 encoder) and 4 buttons for effects. It allows you to switch between controlling all 3 effects of a chain or controlling each parameter of one effect.
+
+By default, 3 knobs are used for controlling effect metaknobs and the buttons under them toggle each effect on/off. To temporarily toggle an effect on/off, press and hold an effect enable button. The 4th knob is used for the mix (dry/wet) knob of the whole chain. On controllers designed for Serato, the :guilabel:`Beats` encoder is used as the mix knob.
+
+The button next to the mix knob is used for focusing effects. On controllers designed for Serato, this button is labeled :guilabel:`Tap`. While holding this button pressed, the enable buttons for each effect switch to choosing which effect is focused. When an effect is focused;
```

Comment:
```suggestion
The button next to the mix knob is used for focusing effects. On controllers designed for Serato, this button is labeled :guilabel:`Tap`. While holding this button pressed, the enable buttons for each effect switch to choosing which effect is focused. When an effect is focused:
```
just amend that change (and other potential small fixes) to the commit and change the commit message "end of file marker?" > "document standart effects mapping"

Reply:
Alright. Let me do that right now.